introduction.md

```diff
@@ -1,3 +1,7 @@
 Here is an introduction of myself
 OK
 https://www.codeschool.com/paths/ruby
+= Deadly skills
+* Git
+* JS
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?